task_manager/Cargo.toml

```diff
@@ -22,9 +22,13 @@ rand = "0.9.0-alpha.1"                                            # This is an a
 rand_chacha = "0.9.0-alpha.1"
 tempfile = "3.10.1"
 alloy-primitives = { workspace = true, features = ["getrandom"] }
-
 rusqlite = { workspace = true, features = ["trace"] }
```

Comment:
This should become an optional feature enabled only when the `sqlite` feature is active

Reply:
Not sure if later we need to use run-time flag or just using compile-time feature to choose the db impl. but yes, should be linked to optional sqlite now.